Unhandled exception executing command 'jobs' in plugin Jobs v4.10.3

Spigotversion: CraftBukkit version git-Spigot-5629554-4c8327f (MC: 1.13.2) (Implementing API version 1.13.2-R0.1-SNAPSHOT)

Jobs Version: Jobs v4.10.3


**After we reset the jobs.sqlite.db have we get this error message all the time. Does one ?? if so, how can it be solved?**


_Woodcutter:
    fullname: Holzfäller
    shortname: W
    description: Verdiene Taler mit Holz fällen und Laub entfernen.
    ChatColour: GREEN
    chat-display: none
    max-level: 200
    leveling-progression-equation: 10*(joblevel)+(joblevel*joblevel*4)
    income-progression-equation: baseincome+(baseincome*(joblevel-1)*0.01)-((baseincome+(joblevel-1)*0.001) * ((numjobs-1)*0.05))
    points-progression-equation: basepoints+(basepoints*(joblevel-1)*0.01)-((basepoints+(joblevel-1)*0.001) * ((numjobs-1)*0.05))
    experience-progression-equation: baseexperience-(baseexperience*((numjobs-1) *0.01))
    rejoinCooldown: 10
    Gui:
      Item: diamond_axe_

```
[12:32:52] [Server thread/INFO]: Starnos issued server command: /jobs join Holzfäller
[12:32:52] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'jobs' in plugin Jobs v4.10.3
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:48) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:704) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1620) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1460) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:9) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_201]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_201]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) [minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:896) [minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831) [minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729) [minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_201]
Caused by: java.lang.IndexOutOfBoundsException: Index: 2, Size: 2
	at java.util.ArrayList.rangeCheck(ArrayList.java:657) ~[?:1.8.0_201]
	at java.util.ArrayList.get(ArrayList.java:433) ~[?:1.8.0_201]
	at com.gamingmesh.jobs.Signs.SignUtil.SignUpdate(SignUtil.java:178) ~[?:?]
	at com.gamingmesh.jobs.PlayerManager.joinJob(PlayerManager.java:359) ~[?:?]
	at com.gamingmesh.jobs.commands.list.join.perform(join.java:89) ~[?:?]
	at com.gamingmesh.jobs.commands.JobsCommands.onCommand(JobsCommands.java:109) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	... 15 more
```

**The jobs are all available. there were no problems before the reset. We have been using JobConfig for 2 years. But now he screams at every job the user accepts. He also gives the jobs and the users have the job then.  This error code comes only if they accept whatever job.**


[16:20:42] [Server thread/INFO]: Starnos issued server command: /jobs join Erbauer
[16:20:42] [Server thread/ERROR]: null
org.bukkit.command.CommandException: Unhandled exception executing command 'jobs' in plugin Jobs v4.10.3
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:48) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:141) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at org.bukkit.craftbukkit.v1_13_R2.CraftServer.dispatchCommand(CraftServer.java:704) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.PlayerConnection.handleCommand(PlayerConnection.java:1620) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.PlayerConnection.a(PlayerConnection.java:1460) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:45) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.PacketPlayInChat.a(PacketPlayInChat.java:1) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.PlayerConnectionUtils.lambda$0(PlayerConnectionUtils.java:9) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:511) [?:1.8.0_201]
	at java.util.concurrent.FutureTask.run(FutureTask.java:266) [?:1.8.0_201]
	at net.minecraft.server.v1_13_R2.SystemUtils.a(SourceFile:199) [minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.MinecraftServer.b(MinecraftServer.java:896) [minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.DedicatedServer.b(DedicatedServer.java:417) [minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.MinecraftServer.a(MinecraftServer.java:831) [minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at net.minecraft.server.v1_13_R2.MinecraftServer.run(MinecraftServer.java:729) [minecraft_server.jar:git-Spigot-5629554-4c8327f]
	at java.lang.Thread.run(Thread.java:748) [?:1.8.0_201]
Caused by: java.lang.IndexOutOfBoundsException: Index: 2, Size: 2
	at java.util.ArrayList.rangeCheck(ArrayList.java:657) ~[?:1.8.0_201]
	at java.util.ArrayList.get(ArrayList.java:433) ~[?:1.8.0_201]
	at com.gamingmesh.jobs.Signs.SignUtil.SignUpdate(SignUtil.java:178) ~[?:?]
	at com.gamingmesh.jobs.PlayerManager.joinJob(PlayerManager.java:359) ~[?:?]
	at com.gamingmesh.jobs.commands.list.join.perform(join.java:89) ~[?:?]
	at com.gamingmesh.jobs.commands.JobsCommands.onCommand(JobsCommands.java:109) ~[?:?]
	at org.bukkit.command.PluginCommand.execute(PluginCommand.java:46) ~[minecraft_server.jar:git-Spigot-5629554-4c8327f]
	... 15 more

--- a/src/main/java/com/gamingmesh/jobs/Signs/SignUtil.java
+++ b/src/main/java/com/gamingmesh/jobs/Signs/SignUtil.java
@@ -175,6 +175,8 @@ public class SignUtil {
 		if (!UpdateHead(sign, PlayerList.get(0).getPlayerName(), timelapse))
 		    timelapse--;
 	    } else {
+	    	if (one.GetNumber() > PlayerList.size())
+	    		continue;
 		TopList pl = PlayerList.get(one.GetNumber() - 1);
 		String PlayerName = pl.getPlayerName();
 		if (PlayerName.length() > 8) {

